components/displayMessages/displayMessages.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { StyledFinalMessagesContainer } from './displayMessages.styles'
+
+type DisplayMessagesProps = {
+  displayMessages: any;
+};
+
+export default function DisplayMessages ({
+  displayMessages
+}: DisplayMessagesProps) {
+  return (
+    <StyledFinalMessagesContainer>
+      {displayMessages()}
```

Comment:
Sorry for this I meant to copy paste 

`        {finalMessages.map((msg) => (
          <div key={nanoid()}>
            <StyledDivider />
            {getErrorMessage(msg.id) && (
              <StyledErrorMessage>
                Error: {getErrorMessage(msg.id)}
              </StyledErrorMessage>
            )}
            <FinalMessage
              id={msg.id}
              to={msg.to}
              subject={msg.subject}
              parentCallback={editFinalMessages}
              content={msg.content}
            />
          </div>
`

out of displayMessages into here

So that function should be removed from the emailSender file and the props passed here should be finalMessages - lmk if u have questions!!

Reply:
ooooops 👀  do i revert what i did?